cloud/aws/templates/aws_oidc/bin/aws_cli.py

```diff
@@ -109,6 +109,23 @@ def wait_for_ecs_service_healthy(self):
             )
             time.sleep(30)
 
+    def fix_digest_value(self, value):
```

Comment:
**suggestion (blocking):** this method is a bit more generic than "fix" since it really just sets the value, recommend renaming it to `set_lock_table_digest_value` and keeping the discussion of why we do this in the pydoc